java/src/main/java/com/powsybl/dataframe/BaseDataframeMapperBuilder.java

```diff
@@ -22,6 +25,7 @@
 
     protected Function<T, List<U>> itemsProvider;
     protected BiFunction<T, String, U> itemGetter;
+    protected NetworkDataframeMapperBuilder.TriFunction<T, UpdatingDataframe, U> itemMultiIndexGetter;
```

Comment:
That class is supposed to be more generic than just for `Network` (for example it's used also for security analysis results): any network specific class should not appear here!